### PR TITLE
fix: added copyright notices for python scripts

### PR DESCRIPTION
**Pull request by reprabhala**
### What changes were proposed in this pull request?

This PR adds copyright licences to all python scripts in the couler project as a fix to the [issue 99](https://github.com/couler-proj/couler/issues/99).

### Why are the changes needed?

Documentation and copyright coverage in all python scripts.
Fixes Issue #99 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

No tests. Changes correspond to copyrights only.
Workflow basic test updated to address test failures after adding the copyright license.

### CODE REVIEW

File: couler/proto/couler_pb2.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The Couler Authors. All rights reserved.
```
Comment (terrytangyuan) [inline couler/proto/couler_pb2.py:1]: This file is autogenerated so we don't need the copyright notice here

File: couler/proto/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 The Couler Authors. All rights reserved.
```
Comment (terrytangyuan) [inline couler/proto/__init__.py:1]: Do you mind updating the year to 2021 in all the files?

**Comment by terrytangyuan** [comment]
Thanks! Please also sign the CLA following the instruction in the comments above.

**Comment by reprabhala** [comment]
@terrytangyuan please let me know if any further changes are required for this PR.

**Review by terrytangyuan — APPROVED** [review]
Thanks!

**Pull request merged**